### PR TITLE
Border sizing and ArrowForward icon

### PR DESCRIPTION
**Pull request by jackrr**
See commits

### CODE REVIEW

File: src/components/spacing.tsx [hunk 1]
```diff
@@ -85,20 +90,21 @@ const TheSpacingContainer = styled.div<SpacingProps>`
...
+  ${({ bb, theme }) =>
+    bb ? borderRule(bb, theme.colors.border, 'bottom') : ''}
+  ${({ bl, theme }) => (bl ? borderRule(bl, theme.colors.border, 'left') : '')}
+  ${({ br, theme }) => (br ? borderRule(br, theme.colors.border, 'right') : '')}
   ${({ by, theme }) =>
     by
-      ? borderRule(theme.colors.border, 'bottom') +
-        borderRule(theme.colors.border, 'top')
+      ? borderRule(by, theme.colors.border, 'bottom') +
+        borderRule(by, theme.colors.border, 'top')
```
Comment (rogervera) [inline src/components/spacing.tsx:102]: Ah, if `by` is falsey,  we still want the default  border size of `1px`.

Comment (jackrr) [reply src/components/spacing.tsx:102]: Hmmm no. If `by` is falsy, we don't want a border. If `by` is true, we want a border of `px`. If `by` is a non-zero number, we want a border of `by`px in stroke.

Comment (jackrr) [reply src/components/spacing.tsx:102]: I think my description above is the behavior this code provides. Do you see something different?

Comment (rogervera) [reply src/components/spacing.tsx:102]: My mistake. I missed the `: ''}` latter part of the ternary. 😄

Comment (jackrr) [reply src/components/spacing.tsx:102]: Ah, cool. Glad it looks right!

**Pull request merged**